services/notify/index.js

```diff
@@ -10,19 +10,29 @@ const notifyClient = new NotifyClient(apiKey);
 function createNotifyService(spec) {
     const {logger} = spec;
     async function sendSms(options) {
-        let response;
         try {
-            response = notifyClient.sendSms(options.templateId, options.phoneNumber, {
-                personalisation: {
-                    case_reference: options.personalisation.caseReference
-                },
-                reference: null
-            });
+            const smsSendRequest = await notifyClient.sendSms(
+                options.templateId,
+                options.phoneNumber,
+                {
+                    personalisation: {
+                        case_reference: options.personalisation.caseReference
+                    },
+                    reference: null
+                }
+            );
+
+            // This DOES NOT indicate that the sms was successfully delivered,
+            // only that the request to send the sms was successful. In future, the id
+            // could be used to further query the status of the request.
+            return {
+                id: smsSendRequest.body.id
+            };
         } catch (err) {
-            logger.error({err}, 'SMS SEND FAILURE');
+            logger.error(err.error, 'SMS SEND FAILURE');
         }
 
-        return response;
+        return undefined;
```

Comment:
would it be worth having the return values in the same shape here, do you think?

Reply:
Possibly. I was just maintaining the current behaviour. Nothing uses it at the moment.